src/cpuinfo_x86.c

```diff
@@ -125,6 +125,41 @@ static bool HasTmmOsXSave(uint32_t xcr0_eax) {
                                MASK_ZMM16_31 | MASK_XTILECFG | MASK_XTILEDATA);
 }
 
+static bool HasSecondFMA(uint32_t model) {
```

Comment:
Can I ask you to run `clang-format` as well?

Reply:
How do I run it on this project?  I could not find an example in the CI system.  I have never run `clang-format` before and the recipe I see in Kokkos generates a larger change than I expected.
```
export CLANG_FORMAT_EXE=/usr/local/Cellar/llvm/10.0.1_1/bin/clang-format
export CLANG_FORMAT_EXECUTABLE=${CLANG_FORMAT_EXE:-clang-format}
find ${TRACKED_FILES}   -type f -name '*.cpp' -o -name '*.hpp' -o -name '*.cc' -o -name '*.h' |   xargs -n 1 -P 10 ${CLANG_FORMAT_EXECUTABLE} -i
```